src/internal/modules/contact-entry/forms/select-account-type.js

```diff
@@ -0,0 +1,50 @@
+const { formFactory, fields } = require('shared/lib/forms');
+const Joi = require('@hapi/joi');
+
+const form = (request, accountTypeDefaultValue, personNameDefaultValue) => {
+  const { csrfToken } = request.view;
+  const { sessionKey } = request.query;
+
+  const f = formFactory('/contact-entry/new/account-type');
+
+  f.fields.push(fields.radio('accountType', {
+    errors: {
+      'any.required': {
+        message: 'Select an option'
+      }
+    },
+    choices: [
+      { value: 'organisation', label: 'Company or organisation' },
+      {
+        value: 'person',
+        label: 'Individual',
+        fields: [fields.text('personName', {
+          errors: {
+            'any.empty': {
+              message: 'Error message goes here'
```

Comment:
Should use a more suitable error message: 'Enter the Individual's full name'

Reply:
Oh god. I'm so sorry. Not sure what happened there. I'm sure I edited it but obviously not. 🙈  Apologies.